lua/refactoring/tests/refactor_spec.lua

```diff
@@ -40,13 +41,29 @@ local function get_refactor_name_from_path(path)
     return refactor_name
 end
 
-local function for_each_file(cb)
+-- Files that don't require lsp to test
+local function for_each_non_lsp_file(cb)
     local files = scandir.scan_dir(
         Path:new(cwd, "lua", "refactoring", "tests"):absolute()
     )
     for _, file in pairs(files) do
         file = remove_cwd(file)
-        if string.match(file, "start") then
+        -- TODO: Should do something better to ignore these cases for future
+        -- refactor operations
+        if string.match(file, "start") and not string.match(file, "123") then
```

Comment:
perhaps move this to a map instead of a string match check for 123

Reply:
Remove this as all tests are under same function (aside from empty input tests).